cloudsensei/README.md

```diff
@@ -0,0 +1,91 @@
+## CloudSensei
+
+CloudSensei is a slack bot integrated with cloud governance, 
+allowing users to join a “read-only” slack channel to review 
+daily expense reports for instances (in running state) >= 7 days.
```

Comment:
Please update to:

CloudSensei is an effort to uncover potential cloud resource leaks leading to inefficient cloud management.  Currently, we are generating a daily report allowing users to action on long-running EC2 instances. 

**To-Do:**
Eventually, CloudSensei will be flipped to work as a Slackbot, allowing users to join a “read-only” Slack channel to review daily expense reports for instance.